hazelcast/src/main/java/com/hazelcast/config/ConfigXmlGenerator.java

```diff
@@ -160,7 +160,8 @@ private static void collectionXmlGenerator(XmlGenerator gen, String type, Collec
                         .node("statistics-enabled", config.isStatisticsEnabled())
```

Comment:
Seems `mapConfigXmlGenerator` is missing the `quorum-ref`.

Reply:
Lol, this is an old structure, so it was never there...

Reply:
Fixed.